assets/js/script.js

```diff
@@ -107,7 +107,7 @@ sectionContainer.append(
 projectData.forEach((element, index) => {
 // Create Speakers Card with img, cardInfo div, h3, h4, line, p
 
-  const speakersCard = document.createElement('div');
+  const speakersCard = document.createElement('article');
   const speakersImg = document.createElement('img');
   const speakersCardInfo = document.createElement('div');
   const speakersh3 = document.createElement('h3');
```

Comment:
- [ ] (OPTIONAL) Great job so far 👍 In the mobile view of the homepage, it would be great if you could display 2 animals together with a `more` button. Then when a user clicks on the `more` button, the rest of the animals are then displayed. This will improve the UX of your site 💯 
- Required design:
![Annotation 2022-06-25 032633](https://user-images.githubusercontent.com/85113567/175754841-7b0dbf63-7858-4728-9161-cfdcd9a5abf4.jpg)